plugin/src/main/resources/data/lang/en.yml

```
@@ -72,4 +72,6 @@ messages:
       notInGame: "{lang:prefix} <#f2b8c6>You are not in a game"
       unknown: "{lang:prefix} <#f2b8c6>An unknown error occurred. Please try again"
     respawn:
-      timeLeft: "<#8bd5ff>{secondsLeft}"
\ No newline at end of file
+      timeLeft: "<#8bd5ff>{secondsLeft}"
+    teamBasedStocks:
+      winner: "{lang:prefix} <#e8edff>Team <#a6e3a1>{winners}</#a6e3a1> <#e8edff>won <#8bd5ff>{lang:minigames.{minigameId}.name}</#8bd5ff>!"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)